.github/pull_request_template.md

```diff
@@ -21,6 +21,10 @@
 - [ ] I've reached out for help from the docs team
 - [ ] No docs needed for this change
 
+## How much of this change is AI generated (0-10)?
+
+<!-- Where 0 is no AI was used at all, and 10 is its all written by AI with no human checking -->
```

Comment:
Scale from 0 to 5 would feel easier to… grok

Extra note: external contributors would benefit from something like "whether you've used any AI or not – you should always first review your change yourself"